TFNetworkRecLayer.py

```diff
@@ -6116,3 +6116,60 @@ def __call__(self, inputs, state, scope=None):
     new_state = tf.nn.rnn_cell.LSTMStateTuple(c, h)
 
     return output, new_state
+
```

Comment:
There must be two empty lines before a new class. (For all code, always, ever: Keep it consistent to the existing code.) (Please use PyCharm, and check for the warnings. PyCharm should have notified about that.)